docker-compose.yml

```diff
@@ -52,6 +53,12 @@ services:
       - /dev/shm:/dev/shm
       - &pg-sock-volume pg-sock:/var/run/postgresql
       - &redis-run-volume redis-run:/var/run/redis
+    shm_size: 1g
```

Comment:
does it fit? this likely includes sklearn and numpy and all that fun stuff, right?

Reply:
This is shared memory space for things like IPC